### PR TITLE
*: the result of `admin show ddl ...` shows a job‘s start time

### PR DESCRIPTION
**Pull request by zimulala**
Remove the useless field of `LastUpdateTS` in Job.
Add the field of `StartTS` and support showing the start time in the result of `admin show ddl ...`.

### CODE REVIEW

File: ddl/ddl_worker.go [hunk 1]
```diff
@@ -124,7 +125,6 @@ func (d *ddl) handleUpdateJobError(t *meta.Meta, job *model.Job, err error) erro
 // updateDDLJob updates the DDL job information.
 // Every time we enter another state except final state, we must call this function.
 func (d *ddl) updateDDLJob(t *meta.Meta, job *model.Job, updateTS uint64) error {
```
Comment (winkyao) [inline ddl/ddl_worker.go]: so updateTS param can be removed?

File: model/ddl.go [hunk 2]
```diff
@@ -193,6 +200,13 @@ func (job *Job) String() string {
 		job.ID, job.Type, job.State, job.SchemaState, job.SchemaID, job.TableID, rowCount, len(job.Args))
 }
 
+// StringWithStartTime returns a job's information with start time.
+// Getting start time tasks time, which impacts the performance.
+// So it's only used to show DDL information.
+func (job *Job) StringWithStartTime() string {
```
Comment (winkyao) [inline model/ddl.go]: Maybe StartTimeInfo() is better?

File: ddl/stat.go [hunk 3]
```diff
@@ -68,7 +68,7 @@ func (d *ddl) Stats(vars *variable.SessionVars) (map[string]interface{}, error)
 	if ddlInfo.Job != nil {
 		m[ddlJobID] = ddlInfo.Job.ID
 		m[ddlJobAction] = ddlInfo.Job.Type.String()
-		m[ddlJobLastUpdateTS] = ddlInfo.Job.LastUpdateTS / 1e9
+		m[ddlJobStartTS] = ddlInfo.Job.StartTS / 1e9
```
Comment (winkyao) [inline ddl/stat.go]: add a comment about the unit of StartTS . so we know why divide it by 1e9.

**Comment by coocood** [comment]
@zimulala 
Why not just modify the `String` function to include the startTS?

**Comment by zimulala** [comment]
I added comments “Getting start time tasks time, which impacts the performance.” So I didn't modify `String` directly.

File: model/ddl.go [hunk 4]
```diff
@@ -193,6 +200,13 @@ func (job *Job) String() string {
 		job.ID, job.Type, job.State, job.SchemaState, job.SchemaID, job.TableID, rowCount, len(job.Args))
 }
 
+// StringWithStartTime returns a job's information with start time.
+// Getting start time tasks time, which impacts the performance.
```
Comment (coocood) [inline model/ddl.go]: Getting startTime is very cheap, it takes just a few nanoseconds.

**Comment by coocood** [comment]
LGTM

**Comment by shenli** [comment]
Will this PR be compatible with the previous DDL job?

**Comment by zimulala** [comment]
Yes.

**Comment by shenli** [comment]
LGTM

**Pull request merged**